thankyou_letters/models/thankyou_config.py

```diff
@@ -36,9 +38,12 @@ def for_donation(self, invoice_lines):
         assert len(self) > 0, "There should be at least one Thank you configuration."
         # Cover the case where the total_amount is smaller that all min_
         # donation amount.
-        config = self[0]
+
+        filtered_config = self.filtered(lambda x: x.lang is False or invoice_lines.partner_id.lang == x.lang)
```

Comment:
If there are multiple invoice_lines this line will fail.
```suggestion
        filtered_config = self.filtered(
            lambda x: x.lang is False or
            invoice_lines.mapped("partner_id").lang == x.lang)
```